pkg/vmtest/integration.go

```diff
@@ -39,11 +40,6 @@ type Options struct {
 	// possible.
 	QEMUOpts qemu.Options
 
-	// DontSetEnv doesn't set the BuildOpts.Env and uses the user-supplied one.
```

Comment:
don't remove this for the moment -- pass it along? u-bmc uses this.

Reply:
Done!